website/source/layouts/aws.erb

```diff
@@ -10,6 +10,15 @@
                     <a href="/docs/providers/aws/index.html">AWS Provider</a>
                 </li>
 
+                <li<%= sidebar_current(/^docs-aws-datasource/) %>
```

Comment:
Yes, the long navigation bar is clearly unsustainable for larger providers. I guess that will be looked at soonish.